utils/get_release_notes.rb

```diff
@@ -0,0 +1,129 @@
+require 'uri'
+require 'net/http'
+require 'openssl'
+require 'json'
+
+PATH_TO_ZENHUB_TOKEN = 'token'
```

Comment:
Can this be an environment variable - so I can set it up once and not fix it on every (fresh) checkout?

Reply:
Could be. @GeorgiChochov  what do you think? Make a token ENV variable?

Reply:
Could actually even use a Github token